process-document/src/main/java/com/ritense/processdocument/service/ProcessDocumentAssociationService.java

```diff
@@ -43,7 +43,7 @@ public interface ProcessDocumentAssociationService {
 
     ProcessDocumentDefinition getProcessDocumentDefinition(ProcessDefinitionKey processDefinitionKey, long documentDefinitionVersion);
 
-    List<? extends ProcessDocumentDefinition> findProcessDocumentDefinitions(String documentDefinitionName);
+    List<? extends ProcessDocumentDefinition> findProcessDocumentDefinitions(String documentDefinitionName, Boolean startableByUser);
```

Comment:
This is a breaking change. This method should be a new one, where the old one calls it with `null`.